### PR TITLE
Fix share option being displayed erroneously

### LINKED ISSUE #117: Share option being displayed erroneously
The share option is displayed on public share pages even when it does not function.

### PR DESCRIPTION
**Pull request by Pytal**
Fix #117

### CODE REVIEW

File: js/files.js [hunk 1]
```diff
@@ -81,7 +81,7 @@
 
             var share = currentFile.find('.filename .fileactions .action-share');
 
-            if (share.length === 0) {
+            if (share.length !== 0) {
                 addNewOption('Share', 'shared', t(appName, 'Share ' + (currentFile.attr('data-type') === 'dir' ? 'folder' : 'file')), function () {
```
Comment (nickvergessen) [inline js/files.js:85]:
```suggestion
                addNewOption('Share', 'shared', (currentFile.attr('data-type') === 'dir' ? t(appName, 'Share folder') : t(appName, 'Share file')), function () {
```

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: js/files.js (hunk 1 above)
Comment (Pytal) [reply js/files.js]: Commited :)

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**